src/compiler/compile.js

```diff
@@ -10,16 +13,16 @@ import {getBuilders} from './builders';
  */
 export function compile(request) {
   const {component_versions: versions, document, nodes} = request ?? {};
-  if (!versions || !(document || nodes)) {
```

Comment:
nit: slightly prefer the early return format from before. only mildly though, this is fine.

Reply:
ack